modules/objdetect/src/face_detect.cpp

```diff
@@ -37,17 +37,24 @@ class FaceDetectorYNImpl : public FaceDetectorYN
         inputW = input_size.width;
         inputH = input_size.height;
 
+        divisor = 32;
+        padW = (int((inputW - 1) / divisor) + 1) * divisor;
+        padH = (int((inputH - 1) / divisor) + 1) * divisor;
+
         scoreThreshold = score_threshold;
         nmsThreshold = nms_threshold;
         topK = top_k;
 
+        strides = {8, 16, 32};
```

Comment:
I would suggest make it static

Reply:
Hello, I don't think changing this variable to static is better for performance.